Demo/DemoViewController.swift

```diff
@@ -10,52 +10,60 @@ import UIKit
 import NetworkHandler
 
 class DemoViewController: UITableViewController {
+    
+    // MARK: - Properties
+    
 	let demoModelController = DemoModelController()
+    private var tasks = [UITableViewCell: URLSessionDataTask]()
 
+    // MARK: - Outlets
+    
 	@IBOutlet var generateDemoDataButton: UIButton!
-	private var tasks = [UITableViewCell: URLSessionDataTask]()
 
+    // MARK: - Actions
+    
+    @IBAction func generateDemoDataButtonPressed(_ sender: UIButton) {
+        sender.isEnabled = false
+        demoModelController.generateDemoData { [weak self] in
+            self?.demoModelController.fetchDemoModels(completion: { [weak self] error in
+                if let error = error {
+                    NSLog("There was an error \(error)")
+                }
+                DispatchQueue.main.async {
+                    self?.tableView.reloadData()
+                    sender.isEnabled = true
+                }
+            })
+        }
+    }
+    
+    @objc func refreshData() {
+        tableView.refreshControl?.beginRefreshing()
+        demoModelController.fetchDemoModels { [weak self] (error) in
+            DispatchQueue.main.async {
+                if let error = error {
+                    let alert = UIAlertController(error: error)
+                    self?.present(alert, animated: true)
+                }
+                self?.tableView.refreshControl?.endRefreshing()
+                self?.tableView.reloadData()
+            }
+        }
+    }
+    
+    // MARK: - VC Lifecycle
+    
 	override func viewDidLoad() {
 		super.viewDidLoad()
-
 		tableView.refreshControl = UIRefreshControl()
 		tableView.refreshControl?.addTarget(self, action: #selector(refreshData), for: .valueChanged)
-	}
-
-	@objc func refreshData() {
-		tableView.refreshControl?.beginRefreshing()
-		demoModelController.fetchDemoModels { [weak self] (error) in
-			DispatchQueue.main.async {
-				if let error = error {
-					let alert = UIAlertController(error: error)
-					self?.present(alert, animated: true)
-				}
-				self?.tableView.refreshControl?.endRefreshing()
-				self?.tableView.reloadData()
-			}
-		}
-	}
```

Comment:
It appears this section has been removed. Why?

Reply:
It wasn't removed, just moved into the Actions tab

Reply:
Oh I see it now! My bad.

Reply:
I figured it out - the indentation was changed from tabs to spaces, so git sees that as a complete removal, which is why this whole section is marked as removed (with the exact same code marked as inserted above). So, please fix this to use tabs for indentation.